charts/prometheus/templates/server/deploy.yaml

```diff
@@ -163,6 +163,10 @@ spec:
     {{- if .Values.server.hostAliases }}
       hostAliases:
 {{ toYaml .Values.server.hostAliases | indent 8 }}
+    {{- end }}
+    {{- if .Values.server.securityContext }}
```

Comment:
`{{- if .Values.server.securityContext }}` is this intentional ?

Reply:
@hectorj2f Nope. Thanks for pointing.. Typo, obviously.